packages/puppeteer-core/src/common/CDPSession.ts

```diff
@@ -0,0 +1,154 @@
+import {ProtocolMapping} from 'devtools-protocol/types/protocol-mapping.js';
+
+import {CDPEvents, CDPSession, CDPSessionEvent} from '../api/CDPSession.js';
+import {assert} from '../util/assert.js';
+
+import {
+  CallbackRegistry,
+  Connection,
+  createProtocolErrorMessage,
+} from './Connection.js';
+import {TargetCloseError} from './Errors.js';
+import {CDPTarget} from './Target.js';
+
+/**
+ * @internal
+ */
+
+export class CDPCDPSession extends CDPSession {
```

Comment:
Use the common prefix for specific `Cdp`/`Bidi` implementations
```suggestion
export class CdpCDPSession extends CDPSession {
```

Reply:
We can do this is another PR. We should consistently use `CDP` since that's how it is everywhere in puppeteer.